Check box

sivaraam opened the pull request:
Added Check-box for selecting hours. It's just a prototype, not the final version.

Commit:
Dummy table content added and table made visible

Commit:
Added Check-box for selecting the hour

 * This checkbox is based on a few CSS hacks
 Credits : Wikipedia Login-page

    https://en.wikipedia.org/w/index.php?title=Special:UserLogin

Commit:
Moved background styles in checked state

  Previously background styles for checked state
  were added and the background size was set to 0.

  As that seemed unnecessary the background styles
  were moved to checked state alone

sirganeshv commented:
*It will be Better if you can include that hover option too (in reference with the link in commit message)
*Why did you include light-style instead of dark-style for radio-button ?

Commit:
Fixed the file names for consistency

Commit:
Added box shadow to make the selction visible

 The box shadows are shown when hovering over
 a check box and for a selected check box
